build.sbt

```diff
@@ -108,6 +108,7 @@ lazy val refinedScalacheck = Def.setting("eu.timepit" %%% "refined-scalacheck" %
 lazy val discipline      = Def.setting("org.typelevel" %%% "discipline-core" % "1.1.2")
 lazy val munit           = Def.setting("org.scalameta" %% "munit" % "0.7.16" % Test)
 lazy val munitDiscipline = Def.setting("org.typelevel" %% "discipline-munit" % "1.0.3" % Test)
+lazy val probably        = Def.setting("com.propensive" %% "probably-cli" % "0.8.0")
```

Comment:
should we enable this dependency only for tests? e.g. `% Test`

Reply:
Yes, I think so. (But try it to make sure...)